lib/model/message.dart

```diff
@@ -125,10 +125,21 @@ class MessageStoreImpl with MessageStore {
     if (message == null) return;
 
     message.flags = event.flags;
+    // According to the API, the content of the message is edited if origContent
+    // is not null.
+    if (event.origContent != null) {
+      message.editState = MessageEditState.edited;
+    }
     if (event.renderedContent != null) {
       assert(message.contentType == 'text/html',
         "Message contentType was ${message.contentType}; expected text/html.");
       message.content = event.renderedContent!;
+      if (
+        !(event.renderingOnly == true
+        || event.userId == null) // TODO(server-5) can remove userId check
```

Comment:
There's a similar fallback on `renderingOnly` above, in `_handleUpdateMessageEventTimestamp`. This is meant to do the exact same thing, so let's make it look as similar as possible.

Ideally it'd live in just one place, but that requires some refactoring that probably isn't worth it and would make the code uglier in a different direction (passing `isRenderingOnly` around? giving up on having these organized as separate methods?).

If the stakes were higher, then I guess the polished solution would be to use `JsonKey.readValue` so that `event.renderingOnly` had the right value from the beginning. But I think that isn't worth it; these should be the only two places we care about this value, and they're almost right next to each other.

Reply:
Maybe we can just extract the fallback to `UpdateMessageEvent` as a getter?

Reply:
Now that we don't rely on the logic to set edit state, there is no need to need the fallback any more.